### PR TITLE
Modernize date kit

### PR DESCRIPTION
**Pull request by leilaweiss**
Modernize Date Kit

### CODE REVIEW

File: app/pb_kits/playbook/pb_date/date.rb [hunk 1]
```diff
@@ -41,38 +29,30 @@ def display_value
...
-      end
-
-      def icon
-        pb_icon = Playbook::PbIcon::Icon.new(icon: "calendar", fixed_width: true)
-        ApplicationController.renderer.render(partial: pb_icon, as: :object)
-      end
+      # def icon
+      #   pb_icon = Playbook::PbIcon::Icon.new(icon: "calendar", fixed_width: true)
+      #   ApplicationController.renderer.render(partial: pb_icon, as: :object)
+      # end
```
[the remark this answers is not in the document]
Comment (leilaweiss) [reply app/pb_kits/playbook/pb_date/date.rb]: this is still a working progress that is why i have not deleted code yet!

File: app/pb_kits/playbook/pb_date/date.rb [hunk 2]
```diff
@@ -2,111 +2,38 @@
 
 module Playbook
   module PbDate
-    class Date < Playbook::PbKit::Base
+    class Date
       include ActionView::Helpers::TagHelper
       include ActionView::Context
+      include Playbook::Props
+      partial "pb_date/date"
```
Comment (xjunior) [inline app/pb_kits/playbook/pb_date/date.rb]: Please, add a space before this call and after the last include

File: app/pb_kits/playbook/pb_date/date.rb [hunk 3]
```diff
@@ -2,111 +2,38 @@
...
-      def display_value
-        case size
-        when "lg"
-          display_value_lg
-        when "sm"
-          display_value_sm
-        else
-          display_value_xs
-        end
+      def xs_date
```
Comment (xjunior) [inline app/pb_kits/playbook/pb_date/date.rb]: Add test to this?

File: app/pb_kits/playbook/pb_date/date.rb [hunk 4]
```diff
@@ -2,111 +2,38 @@
...
+        "#{pb_date_time.to_day_of_week.upcase} &middot; #{pb_date_time.to_month.upcase} #{pb_date_time.to_day}".html_safe
       end
 
-      def kit_class
-        "pb_date_kit"
+      def lg_date
```
Comment (xjunior) [inline app/pb_kits/playbook/pb_date/date.rb]: Add test to this?

File: app/pb_kits/playbook/pb_date/date.rb [hunk 5]
```diff
@@ -2,111 +2,38 @@
...
+        "#{pb_date_time.to_month.upcase} #{pb_date_time.to_day}"
       end
 
-      def to_partial_path
-        "pb_date/date"
+      def sm_date
```
Comment (xjunior) [inline app/pb_kits/playbook/pb_date/date.rb]: Add test to this?

**Pull request merged**